cpp/tests/test_odeseir.cpp

```diff
@@ -362,6 +362,49 @@ TEST(TestSeir, get_flows)
     EXPECT_NEAR(dydt_default[2], 25, 1e-12);
 }
 
+TEST(TestSeir, get_flows_two_agegroups)
+{
+    mio::oseir::Model<double> model(2);
+
+    constexpr double total_first_population = 400;
+    constexpr double total_second_population = 200;
+
+    model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Exposed}]   = 100;
+    model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Infected}]  = 100;
+    model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Recovered}] = 100;
+    model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Susceptible}] =
+        total_first_population - model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Exposed}] -
+        model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Infected}] -
+        model.populations[{mio::AgeGroup(0), mio::oseir::InfectionState::Recovered}];
+    model.populations[{mio::AgeGroup(1), mio::oseir::InfectionState::Exposed}]   = 10;
+    model.populations[{mio::AgeGroup(1), mio::oseir::InfectionState::Infected}]  = 10;
+    model.populations[{mio::AgeGroup(1), mio::oseir::InfectionState::Recovered}] = 10;
+    model.populations[{mio::AgeGroup(1), mio::oseir::InfectionState::Susceptible}] =
+        total_second_population - model.populations[{mio::AgeGroup(1), mio::oseir::InfectionState::Exposed}] -
+        model.populations[{mio::AgeGroup(1), mio::oseir::InfectionState::Infected}] -
+        model.populations[{mio::AgeGroup(1), mio::oseir::InfectionState::Recovered}];
+
+    model.parameters.set<mio::oseir::TimeExposed<double>>(2);
+    model.parameters.set<mio::oseir::TimeInfected<double>>(4);
+    model.parameters.set<mio::oseir::TransmissionProbabilityOnContact<double>>(1);
```

Comment:
These params should be set age specific.
just use a simple for loop.

Reply:
also, u can add the correct number of susceptibles using the  `set_difference_from_group_total `function as shown in https://github.com/SciCompMod/memilio/blob/8958e3159c7cdb64265f93f21d025f76d33a91c0/cpp/examples/ode_seir_ageres.cpp#L50